src/utils/index.ts

```diff
@@ -40,12 +40,23 @@ export const arraysEqual = (a: any[], b: any[]) => {
   return true;
 };
 
-export const roundMoney = (value: number, accuracy = 2) => {
-  const SAFE_ACCURACY = accuracy + 4;
+export const moneyCeil = (value: number, accuracy = 2) => {
+  const SAFE_ACCURACY = accuracy >= 6 ? accuracy + 1 : accuracy + 4;
```

Comment:
(minor) As I see it, calculating of `SAFE_ACCURACY` could be moved to a separate function (used twice here)